cachet_url_monitor/configuration.py

```diff
@@ -158,7 +158,12 @@ def evaluate(self):
         """
         try:
             if self.endpoint_header is not None:
-                self.request = requests.request(
+                if "insecure" in self.endpoint_header:
+                    self.request = requests.request(
+                    self.endpoint_method, self.endpoint_url, timeout=self.endpoint_timeout, headers=self.endpoint_header, verify=False
```

Comment:
This could be simply:
```
self.request = requests.request(
                    self.endpoint_method, self.endpoint_url, timeout=self.endpoint_timeout, headers=self.endpoint_header, verify="insecure" in self.endpoint_header
)
```

Reply:
Apologies, May be I didt get it correctly. Do we want to make "verify="insecure" default ?
To be clear, I am trying to add option to use insecure URL and make this choice optional. 
So adding insecure to yml config in headers.

Reply:
I created this PR #119 to showcase what I meant in this comment. I changed the code a little bit, like moving `insecure` out of the header section as it's not really related to the headers. I moved it to the endpoint level, which makes more sense IMHO. I cherry-picked your commit, so it still has reference to you.

I've added two commits to show before and after. Feel free to take a look.